imports/plugins/core/versions/server/migrations/5_update_defaultRoles_to_groups.js

```diff
@@ -86,6 +72,7 @@ Migrations.add({
 
     // finds all accounts with a permission set and assigns them to matching group
     function updateAccountsInGroup({ shopId, permissions = [], groupId }) {
+      if (!groupId) return null;
```

Comment:
Seems like `return [];` would be better since the result is passed to `.concat` above.

Reply:
Good catch 💯